hera/R/execute.R

```diff
@@ -89,7 +89,7 @@ send_plot <- function(plot) {
 
 # currently not exported, because it is only meant to be called
 # from xeus-r / interpreter::execute_request_impl
-execute <- function(code, execution_counter, silent = FALSE) {
+execute <- function(code, execution_counter, silent = FALSE, eval_env) {
```

Comment:
If we can't release a 0.6.1 of xeus-r, I'd have to do: 

```r
execute <- function(code, execution_counter, silent = FALSE, eval_env = rlang::global_env()) {
```

and this might be rejected for cran policies reasons